### PR TITLE
Feature(GeneralChangesUI): added assets  & UI edits in multiple screens

### PR DESCRIPTION
**Pull request by Benlevy1025**
Added new App icons
Updated Assets

on Map:
made "place info" animation appear faster
**Changed default current location to "Eastern Tel Aviv" instead of "Australia"**

Changed text, colors, alpha, fonts, corner radius and sized on multiple screens(....To be continued)

### CODE REVIEW

File: Dizzy/Screens/Search Place/PlaceSearchVC.swift [hunk 1]
```diff
@@ -46,13 +46,13 @@ final class PlaceSearchVC: ViewController {
         
         closeButton.snp.makeConstraints { make in
             make.topMargin.equalToSuperview().offset(Metrics.doublePadding)
-            make.leading.equalToSuperview().offset(Metrics.doublePadding)
+            make.trailing.equalToSuperview().inset(Metrics.doublePadding)
```
Comment (menesh2) [inline Dizzy/Screens/Search Place/PlaceSearchVC.swift]: please use offset and add -

File: Dizzy/UI Components/PlaceInfoView.swift [hunk 2]
```diff
@@ -138,7 +138,8 @@ class PlaceInfoView: UIView {
         reservationButton.setTitle("RESERVE".localized, for: .normal)
         reservationButton.layer.cornerRadius = reservationButtonHeight/3
         reservationButton.setTitleColor(.white, for: .normal)
-        reservationButton.backgroundColor = UIColor.primeryPurple
+        reservationButton.titleLabel?.font =  UIFont(name: "Avenir-Heavy" , size: 12)
```
Comment (menesh2) [inline Dizzy/UI Components/PlaceInfoView.swift]: please use `Fonts.h11(.bold)`

**Review by menesh2 — CHANGES_REQUESTED** [review]

File: Dizzy/Screens/Search Place/PlaceSearchVC.swift (hunk 1 above)
Comment (Benlevy1025) [reply Dizzy/Screens/Search Place/PlaceSearchVC.swift]: done

File: Dizzy/UI Components/PlaceInfoView.swift (hunk 2 above)
Comment (Benlevy1025) [reply Dizzy/UI Components/PlaceInfoView.swift]: dizzy done

**Review by menesh2 — APPROVED** [review]

**Pull request merged**